src/olympia/editors/helpers.py

```diff
@@ -744,30 +749,28 @@ def get_context_data(self):
 
     def request_information(self):
         """Send a request for information to the authors."""
-        emails = [a.email for a in self.addon.authors.all()]
         self.log_action(amo.LOG.REQUEST_INFORMATION)
         if self.version:
             kw = {'has_info_request': True}
             if not self.addon.is_listed and not self.version.reviewed:
                 kw['reviewed'] = datetime.datetime.now()
             self.version.update(**kw)
-        log.info(u'Sending request for information for %s to %s' %
-                 (self.addon, emails))
-        data = self.get_context_data()
-        subject = u'Mozilla Add-ons: %s %s' % (
-            data['name'], self.version.version if self.version else '')
-        send_mail('editors/emails/info.ltxt', subject,
-                  emails, Context(data),
-                  perm_setting='individual_contact')
+        log.info(u'Sending request for information for %s to authors' %
+                 self.addon)
+        subject = u'Mozilla Add-ons: %s %s'
+        self.notify_email('info', subject, perm_setting='individual_contact')
 
     def send_super_mail(self):
         self.log_action(amo.LOG.REQUEST_SUPER_REVIEW)
         log.info(u'Super review requested for %s' % (self.addon))
         data = self.get_context_data()
-        send_mail('editors/emails/super_review.ltxt',
-                  u'Super review requested: %s' % (data['name']),
-                  [settings.SENIOR_EDITORS_EMAIL],
-                  Context(data))
+        message = loader.get_template(
+            'editors/emails/super_review.ltxt').render(
+            Context(data, autoescape=False))
```

Comment:
This reads weird, I'm often alone with this but I love a chaining kind of style for this:

```python
message = (loader
    .get_template('editors/emails/super_review.ltxt')
    .render(Context(data, autoescape=False)))
```
A bit inspired by how some things are formatted in the JavaScript world and I feel like it reads much easier since arguments are clear and easier to distingquish.